Content.Server.Database/Model.cs

```diff
@@ -322,6 +323,9 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                 .HasForeignKey(w => w.PlayerUserId)
                 .HasPrincipalKey(p => p.UserId)
                 .OnDelete(DeleteBehavior.Cascade);
+
+            modelBuilder.Entity<TextAutomodFilter>()
+                .HasIndex(filter => filter.Id);
```

Comment:
This isn't necessary, it's a primary key.

Reply:
Do I need to mark the Id with an attribute?

Reply:
Fixed in [603bf6e](https://github.com/space-wizards/space-station-14/pull/28570/commits/603bf6e348c7ddb05ff4bb70f16c6155e6fbf687)